pyproject.toml

```diff
@@ -27,18 +27,18 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    "ansicolors==1.1.8",
-    "attrs==21.4.0",
-    "autobahn==21.3.1",
-    "jinja2==3.0.2",
-    "pexpect==4.8.0",
+    "ansicolors>=1.1.8",
+    "attrs>=21.4.0",
+    "autobahn>=21.3.1",
+    "jinja2>=3.0.2",
+    "pexpect>=4.8.0",
     "pyserial-labgrid>=3.4.0.1",
-    "pytest==7.2.2",
-    "pyudev==0.22.0",
-    "pyusb==1.2.1",
-    "PyYAML==5.4.1",
-    "requests==2.26.0",
-    "xmodem==0.4.6",
+    "pytest>=7.2.2",
```

Comment:
7.2.2 is unnecessary strict, the commit message that introduced this mentions `>=7.0.0`, we should fix that.

Reply:
Done.